homeassistant/components/kodi/media_player.py

```diff
@@ -457,70 +359,86 @@ async def async_added_to_hass(self):
             )
         )
 
+    @callback
+    def _on_ws_connected(self):
+        """Call after ws is connected."""
+        self._register_ws_callbacks()
+        self.async_schedule_update_ha_state(True)
+
+    async def _async_ws_connect(self):
+        """Connect to Kodi via websocket protocol."""
+        try:
+            await self._connection.connect()
+            self._on_ws_connected()
+        except jsonrpc_base.jsonrpc.TransportError:
+            _LOGGER.info("Unable to connect to Kodi via websocket")
+            _LOGGER.debug("Unable to connect to Kodi via websocket", exc_info=True)
```

Comment:
Usually we only log at one level for the same message

Reply:
@OnFreund is this error usually recoverable? im going to merge this PR as is after a few more of my own checks but well want to do a followup PR to resolve this and any other code style requests that may clme up in the next few days.

Reply:
yes, that was a remainder from the previous implementation.